book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+
+-   Lack of sufficiently available training on more complex aspects of the ELN solution leading to poor, mis-use, or under-use of ELN features
+
+(rr-rdm-elns-choosing-compare-to-paper)=
+### Always compare to paper notebooks
+
+When picking an ELN remember to always compare the ELN to a paper notebook and not just to other ELNs.
+It is likely paper that you are replacing so this is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
+Consider also which properties of a paper copy is it important to you that you are able to retain when adopting an electronic alternative.
+
+-   A paper lab notebook is physically under your control.
+-   You (or more likely your organisation) own it.
+-   You can control access to it physically.
+-   Your physical possession of this resource means that it would be difficult for anyone to prevent you from accessing it.
+-   Conversely it must be physically stolen to be accessed by a malicious third party.
+-   On the other hand, ELNs - like all software - have security vulnerabilities.
+-   You are not dependent on the functioning of any complex systems like computer networks in order to be able to use your paper lab notebook.
+-   You do not need any specialist tools in order to access its contents.
+-   There is no reason why you might have to pay a third party a fee in order to continue using it.
+-   You do not have to agree to a 'terms of service' or 'end-user license agreement' with a third party to use and retain access to your lab book. These terms may be subject to unilateral alteration by that third party.
+
+If your provider of paper lab books goes out of business it has almost no bearing on your ability to continue doing your work.
+One paper notebook is much like another, finding a new provider is pretty easy.
+Changing providers does not impact on your ability to access your past notebooks or to continue operating with the same workflow in future ones.
+This is not necessarily true of ELNs.
+
+(rr-rdm-elns-choosing-archival)=
+### Archival function of notebooks
+
+Few active measures are needed to maintain the data in your paper notebooks. 
+The primary vulnerability of paper lab books is that there is only one copy. 
+Nevertheless, as long as they are kept in a cool, dry and dark spot (for example a fireproof safe) they will likely last decades.
+
+Electronic data requires much more active upkeep.
+Electronic data can, *if managed properly*, be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive, or even a solid state drive, however, cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bit-rot or compatibility issues when plugged back in.
+
+Lab notebooks perform an archival function.
```

Comment:
```suggestion
For digital data it is important for archiving and maintaining the data to use formats that maximize the likelihood of recovering relevant information.
```
This paragraph starts as something which should probably be the first paragraph of this section, but then only talks about digital/proprietary formats, so I would lean into that second bit and rewrite the first lines.

Reply:
Addressed in [a63fdec](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/a63fdec94d17943207bf2f62169d45151de10099)